tests/pdctl/keyspace/keyspace_group_test.go

```diff
@@ -561,8 +561,13 @@ func TestShowKeyspaceGroupPrimary(t *testing.T) {
 		args := []string{"-u", pdAddr, "keyspace-group"}
 		output, err := pdctl.ExecuteCommand(cmd, append(args, "1")...)
 		re.NoError(err)
+		if strings.Contains(string(output), "Failed") {
+			// It may be failed when meets error, such as [PD:etcd:ErrEtcdTxnConflict]etcd transaction failed, conflicted and rolled back
+			fmt.Println(string(output))
```

Comment:
remove it?

Reply:
I'm concerned about encountering other errors that cause `testutil.Eventually` to keep retrying and eventually fail. Without log output, we may not be able to know what happened.

Reply:
re.NoErrorf(err, "output: %s", string(output))